README.md

```diff
@@ -16,42 +16,38 @@ API service and web client of the Registry of Open Community Challenges
 
 1. Create the file that contains the future environment variables
 
-       cp .env.sample .env
+       cp .env.example .env
 
-2. Export the variables defined in *.env* to environment variables
-
-       export $(grep -v '^#' .env | xargs -d '\n')
-
-3. Start the Data Node API service
+2. Start the Data Node API service
 
-       docker-compose up
+       docker-compose up --build
 
-4. Go to the API service documentation page <http://localhost:8080/api/v1/ui/>
+3. Go to the API service documentation page <http://localhost:8080/api/v1/ui/>
```

Comment:
I think we should also add a step after this on how to stop the service:

"To stop the API service, enter `Ctrl+C` followed by `docker-compose down`.  If running in detached mode, you will only need to enter `docker-compose down`."

Reply:
Done in next commit.